### PR TITLE
[FLOC-3956] Enable Travis CI for dvol

### PR DESCRIPTION
**Pull request by exarkun**
Fixes https://clusterhq.atlassian.net/browse/FLOC-3956

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    # a Travis CI-native way to do pip caching but I don't quite understand it.
+    - "$HOME/.cache/pip"
+
+before_install:
+  # Get packaging tools that can use and cache wheels.
+  - "pip install --upgrade pip>=7"
+  - "pip install wheel"
+
+install:
+  - "pip install ."
```
Comment (Azulinho) [inline .travis.yml]: I haven't tested this, but according to:
 http://stackoverflow.com/questions/19422229/how-to-cache-requirements-for-a-django-project-on-travis-ci

it looks that you may need:
--download-cache $HOME/.pip-cache

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  email: false
+  slack:
+    # Slack integration.
+    # Generated via `travis encrypt -r ClusterHQ/dvol <slack configuration>'
+    # See Travis in the Slack App directory for details.
+    secure: "IuMdO5YgVlYdq12I8PbrnjgQo2mpW53LoABUHoHFnaczpXmOD0cbgSX4YRRqGPTkBj3iMqOYtGbOtcyIRwLB9SVbr2xeDpvRYC8auX/jGMbQmaQMhG4kZOpVA9FKt+XcJFj6lsihT50tx2uJ4zn85KjOy1ZJQjgpka+vsKJbvGAg65AvCovlB/l1fyd8eaRZxDXyvapD7I39R7ij22r/QfnJ0t2JT0PKS0SR+gPemnjMhGkW0n3dyoOOkhqVAEyevt6f+s8NNnSioCuJy6X9R6eIXOeJYXHMvigxpZ7nEnrWik3IgQ9+rqJjjqzb9paxU+1+aQP0gb+r1M+hfbGyWQp1k6KCNb1IljCtjNccWxlajkooLJS1haRKiyV96U+DiDybpVIbEdtNMBQrRN+6lCbU8OduHxluH5WnAdZEUGz6pQLt7fxCazKzib/vxnUlSgvRnKzweYd21zj3c4w6aPLdCaI2eQM036Zwt/9ltWbk5I8sjAWfCgvWqxD+MaOT2vAog8T7B8WoyRjhbChtMNNHtw64a9W402WADrQO1jTU4tGsKwDbjXb4j2GtvhKahrGGigFYBfPLgOi8mzzwaJh7uBo/S8evAG/K/abc2hhhLBGWlSiA252hbNcgjGk8QuwKFhaA1DenAXZTf8GcwYC+ZlmKOczIujamQKfAvjM="
+
+language: python
+python: 2.7
+sudo: false
```
Comment (Azulinho) [inline .travis.yml]: This will run in a docker container, not an OpenVZ container.
hopefully that's what you're looking for

**Comment by Azulinho** [comment]
address the above if you wish, otherwise just berge it.

**Comment by exarkun** [comment]
Thanks for the review!  Regarding pip caching, the caching directory here seems to be correct for the newer version of pip.  And the Docker vs OpenVZ stuff hopefully makes no difference (but maybe we'll just have to find out).  Merging.

**Pull request merged**